wecs/panda3d/character.py

```diff
@@ -182,15 +188,14 @@ def add_shape(self, entity, movement, solid, shape):
         solid['node'] = node
         node.node().add_solid(shape)
         node.node().set_into_collide_mask(0)
-        node.reparent_to(model.node)
+        node.reparent_to(entity[Model].node)
```

Comment:
Please don't. As that component does get used only once, I can see the point, but this is minimal optimization at the cost of readability.